### PR TITLE
Update gif with new logo

### PR DESCRIPTION
**Pull request by ssnl**
preview: https://github.com/SsnL/pytorch/blob/9b6a1183cf639e7006183833d873d91ca54a8d7b/docs/source/_static/img/dynamic_graph.gif

### CODE REVIEW

**Comment by zou3519** [comment]
How are you calling `.backward()` if none of the tensors require grad?

**Comment by SsnL** [comment]
lol @zou3519 wrong thread maybe? ghninja?

**Review by ezyang — APPROVED** [review]

**Comment by zou3519** [comment]
In the gif, none of the tensors require grad, but `loss.backward()` gets called.

**Comment by SsnL** [comment]
@zou3519 Oh I see. Yeah that's how the old gif was. Do you think we should make them require_grad?

**Comment by zou3519** [comment]
I'm not sure if someone's going to try to run the code as-is without any context, but it might be nice

**Comment by soumith** [comment]
yea, we should make them have `requires_grad`. 0.1.9 I think had `requires_grad=True` by default.

**Comment by SsnL** [comment]
Added `requires_grad=True` on the weights. Let me know if this looks good.
